draft-ietf-httpbis-digest-headers.md

```diff
@@ -921,6 +921,37 @@ Digest: id-sha-256=BZlF2v0IzjuxN01RQ97EUXriaNNLhtI8Chx8Eq+XYSc=
 Note that a `204 No Content` response without a payload body 
 but with the same `Digest` field-value would have been legitimate too.
 
+## Error responses
+
+In error responses, the representation-data does not necessarily refer to
+the target resource.
+Instead it refers to the representation of the error.
+
+In the following example, the `Digest` of the response
+is computed on the enclosed representation.
```

Comment:
Lets describe the example a little more, something like the following would help me understand better e.g.

In the following example a client attempts to patch the resource located at /books/123. However, the resource does not exist and the server generates a 404 response with a body that describes the error in accordance with {{?RFC7807}}. The digest of the response is computed on this enclosed representation.

Reply:
thx++. fixed